### PR TITLE
[DEVOPS-384] harmonise microservices

### PR DESCRIPTION
**Pull request by rmharrison**
Harmonisation between microservices.

Refer to `git log` for specific changes
Importantly, after application of DEVOPS-384, repos should be consistent.

### CODE REVIEW

**Comment by rmharrison** [comment]
@ElijahGeorge 

One of the linting notifications...
```
/home/work/git/amida-tech/amida-notification-microservice/server/helpers/pushNotificationHelper.js (2/5)
  ✖  158:17  Assignment to property of function parameter 'data'  no-param-reassign
  ✖  159:17  Assignment to property of function parameter 'data'  no-param-reassign
```

Here: https://github.com/amida-tech/amida-notification-microservice/commit/d13ae91b0ecdc53493e86c2691d09c100fe0d7c2

For why re-assignment can cause bugs, see: https://spin.atomicobject.com/2011/04/10/javascript-don-t-reassign-your-function-arguments/

TL;DR The `data` properties (specifically `data.body` and `data.title`) `deliverNotifications()` uses depends on execution path, which would make it difficult to debug.

What are your thoughts for resolving?

**Comment by jmfadeley** [comment]
Tests check out. Code looks good. I don't know enough of notifications to make a deeper judgment, but it looks good.
